app/templates/view_agreements_g8.html

```diff
@@ -0,0 +1,48 @@
+{% import "toolkit/summary-table.html" as summary %}
+
+{% extends "_base_page.html" %}
+
+{% block breadcrumb %}
+  {%
+      with items = [
+          {
+              "link": url_for('.index'),
+              "label": "Admin home"
+          }
+      ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+  {% with smaller = True, heading = "{} framework agreements".format(framework.name) %}
+    {% include "toolkit/page-heading.html" %}
+  {% endwith %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <div class="marketplace-paragraph">
+        <p>
+          Review the information for each agreement, accept it, and continue to the next one.
+        </p>
+        <p>
+          If there’s a problem with an agreement, contact the person who uploaded it and come back to it later. Ask
+          suppliers to resubmit from the link in the email they received after they were accepted on to the framework.
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div class="grid-row">
+      <section class="column-one-third search-page-filters">
+
+      </section>
+      <section class="column-two-thirds" aria-label="Framework agreements">
```

Comment:
I don't think we need the `aria-label` attribute here because the search results are the main content in the page. 

To explain, I added it to the G-Cloud search page because the search results followed the filters in the page order, meaning they took a while to get to if going through the page in a linear fashion. Giving the search results region an `aria-label` meant they had a human-readable name to identify them so users of assistive tech' could find and jump to them easily.

We don't have the same problems in this page so I think the attribute may confuse matters.